netkan/Dockerfile

```diff
@@ -8,12 +8,13 @@ RUN python -m unittest -v
 WORKDIR /
 USER root
 RUN rm -Rf /netkan
+ENTRYPOINT [".local/bin/netkan"]
+CMD ["--help"]
 
 FROM python:3.7 as dev
 RUN useradd -ms /bin/bash netkan
 ADD . /netkan
 WORKDIR /home/netkan
-ADD run_dev.sh /usr/local/bin/
 ENV PATH "$PATH:/home/netkan/.local/bin"
```

Comment:
We should restore the `run_dev.sh` and the dev `ENTRYPOINT` that was dropped during the VS Code stuff.